packages/sdk-v2/package.json

```diff
@@ -0,0 +1,46 @@
+{
+  "name": "sdk-v2",
```

Comment:
looks like this is a package, so `@elementfi/sdk-v2`, also we need to remove package-lock.json from this package.